packages/python-google-compute-engine/google_compute_engine/instance_setup/instance_setup.py

```diff
@@ -28,8 +28,7 @@
 from google_compute_engine import logger
 from google_compute_engine import metadata_watcher
 from google_compute_engine.boto import boto_config
-from google_compute_engine.compat import urlerror
-from google_compute_engine.compat import urlrequest
+from google_compute_engine.compat import distro_name, urlerror, urlrequest
```

Comment:
Per the style we've been following throughout this codebase, please put these on separate lines.